coral/src/app/layout/Layout.tsx

```diff
@@ -14,7 +14,7 @@ function Layout({ children }: { children: JSX.Element }) {
         minHeight={"screen"}
         style={{
           gridTemplateColumns: "240px 1fr",
-          gridTemplateRows: "auto 1fr",
+          gridTemplateRows: "64px 1fr",
```

Comment:
The layout was broken for `/index` with the way it was implemented before, also I'm still not really sure why. I'm not super experience with building grid layouts, especially with the DS components, so that's what I found out for a fix. 

I can try to see if I find an alternative. Alternative would be to at least use a variable for the height to remove the magic number.

Reply:
In which way was it broken? When you say `index`, you mean the page rendering the filters + topics table?

I remember seeing the layout shift dramatically when the table rows changed, in a past state of the branch, like this:

https://user-images.githubusercontent.com/20607294/206214699-6d2f0ecd-1045-47fb-ae5d-ce1f09e0cbc2.mov

But I seem to remember it was because the rule was like this (which means the content row will have a variable hight given its content):

```tsx
gridTemplateRows: 1fr auto;
```

I don't see such a behaviour with this (`PageHeader`, the first row, has a set height so `auto` will be the same as this set height)

```tsx
gridTemplateRows: auto 1fr;
```

But maybe you are talking about another breakage ^^